dts/arm/nordic/nrf52840.dtsi

```diff
@@ -174,6 +174,22 @@
 			interrupt-names = "wdt";
 			label = "WDT";
 		};
+
+		cryptocell: crypto@5002a000 {
+			compatible = "nordic,nrf-cc310";
+			reg = <0x5002A000 0x1000>;
+			label = "CRYPTOCELL";
+			status = "ok";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			cc310: crypto@5002b000 {
+				compatible = "arm,cryptocell-310";
+				reg = <0x5002B000 0x1000>;
+				interrupts = <42 1>;
```

Comment:
Isn't this interrupt referring, better, to the parent object? Or do I miss anything?

Reply:
what do you mean?

Reply:
>	interrupts = <42 1>;

As far as I understand, this defines the interrupt line. Doesn't this belong to the 

> cryptocell: crypto@5002a000 {

object?

Reply:
No, I think it belongs where it is, since I assume the interrupt is defined by the base cryptocell IP and not nordic specific wrapper/changes.

Reply:
I looked into the nRF52840 product specification, available at the Nordic Infocenter, regarding cryptocell; it does not seem the existing hardware can, actually, generate interrupts.

I would, then, suggest, not to add this info in DTS.
@nordic-doki could you double-check this?

Reply:
The interrupt is correct and it works correctly. Probably it is missing in docs.